backend/package.json

```diff
@@ -28,8 +28,10 @@
     "generate:prisma": "prisma generate",
     "kafka-status": "node ./dist/bin/kafkaStatus.js",
     "seed": "node ./dist/bin/seed.js",
+    "seedAll": "npm run seed && npm run seedPoints && npm run seedExercises && npm run seedCompletions",
```

Comment:
Might want to add these to the "first time run"; also mention that you need to do `npm run build` for these to work (or does it create dist with `npm run dev`, can't remember)

Reply:
Good point, added